### DISCUSSION TITLE
The role of tags

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [buttondown/discussions#24] [Q&A] [answered]
I need to be clear about the role of tags. From what I can tell tags are for organizing subscribers, not tagging content.

**Comment by jmduke** [comment, marked as answer]
Yup, that's right! They're just for segmentation (you can read a bit more about that here: https://docs.buttondown.email/advanced-features/tags). We don't have any CMS-ish features for tagging/taxonomizing content at the moment.

**Reply by jdhitsolutions** [reply]
Ok then. I'll continue adding that to my own content indexing solution.
